common/test_rescue.yml

```diff
@@ -11,6 +11,13 @@
   ansible.builtin.set_fact:
     timestamp: "{{ lookup('pipe', 'date +%Y-%m-%d-%H-%M-%S') }}"
 
+- name: "Set fact of current_testcase_name"
+  ansible.builtin.set_fact:
+    current_testcase_name: "{{ ansible_play_name }}"
+  when: >
+    (current_testcase_name is undefined) or
+    (not current_testcase_name)
+
```

Comment:
Please update `linux/memory_hot_add_basic/memory_hot_add_basic.yml` not this file. You should move up `test_setup.yml` to be executed at first.